ICSharpCode.ILSpyX/MermaidDiagrammer/ReadMe.md

```diff
@@ -0,0 +1,197 @@
+﻿<!-- title: Generating diagrammers --> <!-- of the printed HTML see https://github.com/yzhang-gh/vscode-markdown#print-markdown-to-html -->
+# Generating diagrammers <!-- omit in toc -->
+
+An automated documentation tool for visually exploring
+[.NET assemblies](https://learn.microsoft.com/en-us/dotnet/standard/assembly/) (_*.dll_ files)
+along type relations using rapid diagramming.
+
+<img src="./html/netAmermaid.ico" align="right" title="" />
+
+<!--TOC-->
+- [What can it do for you and how?](#what-can-it-do-for-you-and-how)
```

Comment:
That was my earlier reference of "end user documentation in the Wiki" - basically this document => Wiki.

Reply:
Got you. I understand you mean a new top-level page in https://github.com/icsharpcode/ILSpy/wiki

Reply:
Yes - maybe simply call it Diagramming

Reply:
I've cleaned and split the doco up further
- moving the ilspycmd examples into the ilspycmd Readme (after the help output) and
- extracting the _How does it work?_ chapter into a developer-facing readme
- renaming the leftovers to Wiki.md (because that's where they'll go)

Should `html/UsingDiagrammer.md` remain a standalone page at this point (and if so, where should it go) or be merged into Wiki.md?

Reply:
Is Wiki.md to be supposed in the Wiki or in the repo?

Reply:
`Wiki.md` (and maybe `html/UsingDiagrammer.md` ?) are supposed to go into the Wiki, not the repo.

Reply:
I.e. the blue marked docs are end-user facing and intended to become separate or a single Wiki page (depending on you), the others are dev-facing (or related to ilspycmd parameters and examples) and stay in the repo. Let me know what you think.
![image](https://github.com/user-attachments/assets/5d0a702d-b09f-4406-a4e2-8b7aac21dc38)

Reply:
Yes, end-user facing => Wiki, dev docs along side code please.

Reply:
done.